mreg_cli/utilities/api.py

```diff
@@ -295,7 +295,7 @@ def _request_wrapper(
         result.status_code == 500
         and (operation_type == "post" or operation_type == "patch")
         and params == {}
-        and data is not None
+        and data
```

Comment:
This snuck in while fixing pyright errors, but it's nice to finally silence that error, so we should keep it in.

Reply:
Works for me.